cmd/draft/logs.go

```diff
@@ -51,13 +56,17 @@ func (l *logsCmd) complete(args []string) error {
 	if err := validateArgs(args, l.args); err != nil {
 		return err
 	}
-	l.appName = args[0]
-	l.buildID = args[1]
+	l.buildID = args[0]
 	return nil
 }
 
 func (l *logsCmd) run() error {
-	b, err := l.client.GetLogs(context.Background(), l.appName, l.buildID, draft.WithLogsLimit(l.logLines))
+	deployedApp, err := local.DeployedApplication(draftToml, l.runningEnvironment)
```

Comment:
Maybe we should promote the runningEnvironment flag to the global set? It seems to be common to all subcommands.